spec/features/user_add_items_to_carts_spec.rb

```diff
@@ -2,12 +2,17 @@
 
 RSpec.feature "UserAddItemsToCarts", type: :feature do
   before(:each) do
-    Food.create(
+    F1 = Food.create(
       name: "Spagetti bolognese",
       description: "Warm yummy spagetti to increase your life span!",
-      price: 1500
-    )
-  end
+      price: 1500,
+      prep_time: 12)
+
+    O1 = Order.create(Status: "Completed", total: 23600,
+                   vat: 2100, delivery_cost: 500, user_id: 6, 
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Trailing whitespace detected.